projects/fhi-angular-highcharts/src/lib/services/diagram-type-group.service.ts

```diff
@@ -227,10 +228,20 @@ export class DiagramTypeGroupService {
       this.diagramTypeDisabledWarnings.columnAndLine =
         this.diagramTypeDisabledWarningsText.notTwoUnits;
       return true;
+    } else if (this.uniqueUnitIdCountInSeries() < 2) {
+      this.diagramTypeDisabledWarnings.columnAndLine =
+        this.diagramTypeDisabledWarningsText.notMinTwoUnitsInSeries;
+      return true;
     }
     return false;
   }
 
+  private uniqueUnitIdCountInSeries() {
+    const ids = [];
```

Comment:
nitpicking. It would be easier for me to read if 'ids' was a set in the first place, instead of first creating an array and then convert it to a set. 
```ts
    const uniqueIds = new Set<string | number>();
    this.series.filter((serie) => serie.unitId).forEach((serie) => uniqueIds.add(serie.unitId));
    return uniqueIds.size;
```